renderdoc/driver/gl/wrappers/gl_emulated.cpp

```diff
@@ -1218,6 +1218,70 @@ void APIENTRY _glClearBufferData(GLenum target, GLenum internalformat, GLenum fo
 
 #pragma endregion
 
+#pragma region GLES Compatibility
+
+void APIENTRY _glGetTexImage(GLenum target, GLint level, GLenum format, GLenum type, void *pixels)
+{
+  if((format == eGL_DEPTH_COMPONENT && !HasExt[NV_read_depth]) ||
+     (format == eGL_STENCIL && !HasExt[NV_read_stencil]) ||
+     (format == eGL_DEPTH_STENCIL && !HasExt[NV_read_depth_stencil]))
+  {
+    // return silently, check was made during startup
```

Comment:
This path will have to be implemented eventually one way or another. We need to be able to read back any type of texture for correctness (initial states) but also for remote replay. To display texture data on the PC running the UI we have to read it back on the target to transfer over the network. Without this path we won't be able to display depth textures which is a pretty big problem.

Unfortunately looking around it seems like GL only defines two ways to read back from a texture - glGetTexImage and glReadPixels, so if we're emulating one with the other that kind of limits the options...  I think the next thing would be to add a drawcall in here to sample from depth and render to a temporary 'color' target that has the same pixel size. Is there a better way to approach it? Obviously this doesn't have to be tackled all at once right now - I just want to have an idea of where this is going so we don't lead ourselves up the garden path.